docs/admin/migration.md

```diff
@@ -0,0 +1,15 @@
+# Migrating from KFServing
+
+This doc helps to migrate existing inference services running in the cluster from `KFServing` to `KServe` without downtime. The script deletes your `KFServing` installation after migrating the inference services from `serving.kubeflow.org` to `serving.kserve.io`.
```

Comment:
Can we add an option if user wants to delete the kfserving install or not after the migration ? also can we check if all the previous inference services are fully migrated before the deletion?